iptables.fc: Add missing legacy-restore entries

HuaxinLu opened the pull request:
/usr/sbin/arptables-restore is miss labeled now. It is a link file that can link two differenet file.

On fc 34:

Remove iptables-nft and install ebtables-legacy:
ll /sbin/ebtables-restore 
lrwxrwxrwx. 1 root root 34 Apr 23 06:56 /sbin/ebtables-restore -> /etc/alternatives/ebtables-restore
ll /etc/alternatives/ebtables-restore
lrwxrwxrwx. 1 root root 33 Jun 10 20:31 /etc/alternatives/ebtables-restore -> /usr/sbin/ebtables-legacy-restore

Remove ebtables-legacy and install iptables-nft:
ll /sbin/ebtables-restore
lrwxrwxrwx. 1 root root 34 Apr 23 06:56 /sbin/ebtables-restore -> /etc/alternatives/ebtables-restore
ll /etc/alternatives/ebtables-restore
lrwxrwxrwx. 1 root root 30 Jun 10 20:35 /etc/alternatives/ebtables-restore -> /usr/sbin/ebtables-nft-restore
ll /usr/sbin/ebtables-nft-restore
lrwxrwxrwx. 1 root root 17 Jan 28 08:48 /usr/sbin/ebtables-nft-restore -> xtables-nft-multi

And the label of /usr/sbin/ebtables-legacy-restore is lack.

HuaxinLu committed:
iptables.fc: Add missing legacy-restore and legacy-save entries

/usr/sbin/ebtables-restore and /usr/sbin/ebtables-save are miss labeled now. Each of them is a link file that can link to two differenet files.

For /usr/sbin/ebtables-restore on fc 34:

Remove iptables-nft and install ebtables-legacy:
lrwxrwxrwx. 1 root root 34 Apr 23 06:56 /sbin/ebtables-restore -> /etc/alternatives/ebtables-restore
lrwxrwxrwx. 1 root root 33 Jun 10 20:31 /etc/alternatives/ebtables-restore -> /usr/sbin/ebtables-legacy-restore

Remove ebtables-legacy and install iptables-nft:
lrwxrwxrwx. 1 root root 34 Apr 23 06:56 /sbin/ebtables-restore -> /etc/alternatives/ebtables-restore
lrwxrwxrwx. 1 root root 30 Jun 10 20:35 /etc/alternatives/ebtables-restore -> /usr/sbin/ebtables-nft-restore
lrwxrwxrwx. 1 root root 17 Jan 28 08:48 /usr/sbin/ebtables-nft-restore -> xtables-nft-multi

/sbin/ebtables-save is similar. But the label of /usr/sbin/ebtables-legacy-restore and /usr/sbin/ebtables-legacy-save is lack.

zpytela commented:
@HuaxinLu - confirmed; while there, can you also add label for /usr/sbin/ebtables-legacy-save?

HuaxinLu commented:
> @HuaxinLu - confirmed; while there, can you also add label for /usr/sbin/ebtables-legacy-save?

Of course, I have added it and fixed the pr comment.

zpytela commented:
Merging, thank you for your contribution.
